xcode/X16/iOS/GameControlViewController.xib

```diff
@@ -1,99 +1,99 @@
-                    <state key="normal" title="Select"/>
-                    <connections>
-                        <action selector="buttonPressed:" destination="-1" eventType="touchUpInside" id="bL9-Vl-BRe"/>
-                    </connections>
-                </button>
-                <button opaque="NO" tag="40" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="hCE-KR-RYd">
-                    <rect key="frame" x="22" y="9" width="34" height="30"/>
-                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
-                    <state key="normal" title="Start"/>
-                    <connections>
-                        <action selector="buttonPressed:" destination="-1" eventType="touchUpInside" id="nHb-3i-8Ie"/>
-                    </connections>
-                </button>
-                <button opaque="NO" tag="226" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="aEC-fb-eLv">
-                    <rect key="frame" x="80" y="55" width="30" height="30"/>
-                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
-                    <state key="normal" title="B"/>
-                    <connections>
-                        <action selector="buttonPressed:" destination="-1" eventType="touchUpInside" id="mrO-no-NF6"/>
-                    </connections>
-                </button>
-                <button opaque="NO" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="DHN-U6-bMb">
-                    <rect key="frame" x="278" y="0.0" width="39" height="30"/>
-                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
-                    <state key="normal" title="Close"/>
-                    <connections>
-                        <action selector="closeWindow:" destination="-1" eventType="touchUpInside" id="Ia6-iX-Lrk"/>
-                    </connections>
-                </button>
-            </subviews>
-            <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
-            <freeformSimulatedSizeMetrics key="simulatedDestinationMetrics"/>
-            <viewLayoutGuide key="safeArea" id="Q5M-cg-NOt"/>
-            <point key="canvasLocation" x="98" y="-4"/>
-        </view>
-    </objects>
+	<device id="ipad12_9" orientation="landscape" layout="fullscreen" appearance="light"/>
```

Comment:
.xib with tabs? I'm sure Xcode will change it into spaces again the next time it saves it.

Reply:
If you're happy with me adding them to the editor config then I'll do so.